Transaction/DLC_Contract_Execution_Transaction.md

```diff
@@ -0,0 +1,138 @@
+# DLC Contract Execution Transaction
+
+This details the exact format of the on-chain contract execution transaction (CET), which both sides need to agree on to ensure signatures and adaptor signatures are valid.
+
+# Table of Contents
+
+* [Transaction](#transaction)
+  * [CET Outputs](#cet-outputs)
+    * [Offerer Output](#offerer-output)
+    * [Accepter Output](#accepter-output)
+* [Fees](#fees)
+* [Test Vectors](#test-vectors)
+* [References](#references)
+* [Authors](#authors)
+
+# Transaction
+
+* version: 2
+* locktime: `contract_maturity`
+* txin count: 1
+  * `txin[0]` outpoint: `txid` of funding transaction and `output_index` 0
+  * `txin[0]` sequence: 0xFFFFFFFE
+  * `txin[0]` script bytes: 0
+  * `txin[0]` witness: `0 <signature_for_offer_pubkey> <signature_for_accept_pubkey>`
+
+The output script public keys and `contract_maturity` are negotiated in the offer and accept messages.
+
+There will be one CET for every possible outcome, the output values correspond to such an outcome and are negotiated in the offer message.
+
+## CET Outputs
+
+The Offerer/Initiator's output comes first, the Accepter's output comes second.
+
+Note that this will likely change in the future.
+
+If either party receives less than the dust limit of `1000 satoshis` for this outcome, then their output is not produced.
```

Comment:
Should note that both of these are likely to change

Reply:
Also, I think this is another good case for where we could use bip-69 for input/output ordering for v0.1